files/SZWESAMP/ZWEKRING

```diff
@@ -358,15 +358,17 @@ $$
           ACCESS(CONTROL) 
 
 /* Uncomment this command to allow other user to access key ring ... */
-/* PERMIT &ZOWEUSER..&ZOWERING..LST CLASS(RDATALIB) ID(<user>) +
-/*        ACCESS(CONTROL) 
+/* PERMIT &ZOWEUSER..&ZOWERING..LST CLASS(RDATALIB) ID(<user>) +     */
+/*        ACCESS(READ)                                               */
 
 /* Refresh to dynamically activate the changes. .................... */
    SETROPTS RACLIST(RDATALIB) REFRESH
 
-/* ALTERNATIVE to using RDATALIB profiles ...........................*/
+/* permit server user ID to access certificates .....................*/
```

Comment:
RACF, and I assume ACF2 & TSS too, will first look for the RDATALIB profile. Only if RDATALIB check produces an undetermined answer does RACF check the FACILITY profiles. Since we explicitly define the RDATALIB profile, there will never be an undetermined answer and thus the FACILITY profiles are never checked. 
So while it does not hurt to have them (they are not used), it will look weird to security admins who know the logic. That is why in the sample I gave you the FACILITY profiles were commented out and marked as alternative, since this older setup is still valid, and some customers might still rely on it (in which case they would remove the RDATALIB-specific steps before submitting).

If you decide to leave the FACILITY profiles in place, I advise to change the leading comment as the current one is not correct with RDATALIB in place, Something like
```
IRR.DIGTCERT logic pre-dates RDATALIB logic, and is not used when the RDATALIB profile is used. The following PERMIT commands are present for customers who do not wish to use RDATALIB and want to continue using their existing IRR.DIGTCERT setup. Note that the PERMIT commands will fail if the IRR.DIGTCERT profiles do not already exist.
```